cmd/scheduled-feed/main.go

```diff
@@ -97,8 +98,25 @@ func main() {
 		pub:       pub,
 		delta:     delta,
 	}
+
+	if appConfig.Timer {
+		cronjob := cron.New()
+		crontab := fmt.Sprintf("@every %s", delta.String())
```

Comment:
Could avoid declaring the variable `crontab` and include the `Sprintf` in the log.

Reply:
The var is also used as a parameter the cron `AddFunc()` call else I would agree that it's unnecessary to declare it just for the log